terraform/modules/firewall-policy/variables.tf

```diff
@@ -28,4 +28,8 @@ variable "rules" {
 variable "tags" {
   description = "A map of keys and values used to create resource metadata tags"
   type        = map(any)
+}
+
+variable "list_of_allowed_domains" {
```

Comment:
Could we use `fw_allowed_domains` to stay consistent with the naming of the other variables? We should probably have a `description` in the variable block too.